csfieldguide/chapters/content/en/coding-encryption/sections/the-key-distribution-problem.md

```diff
@@ -89,10 +89,31 @@ For this section on public key systems, we will use RSA as an example.
 
```

Comment:
1) I don't think it matters too much, PKCS is too complicated to explain (hence why I added the components form). The previous interactive ignored it entirely and even removed the headers so you didn't know which format it was.

2) Already explained in the very bottom curiosity panel

3) ??? Perhaps I'll say this doesn't resolve #1081 because I don't know enough about how the padding works

Reply:
1. I don't mean explaining the formats themselves. Maybe just a small sentence mentioning what the differences are? Otherwise why add different formats at all? Maybe Tim can come up with something short.

2. Sorry didn't see!

3. Alright. Maybe Tim could come up with something?